src/test/java/lotto/domain/StatisticsTest.java

```diff
@@ -0,0 +1,25 @@
+package lotto.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+@DisplayName("도메인 Statistics 테스트")
+class StatisticsTest {
+
+    @DisplayName("수익률 산출 테스트")
+    @Test
+    void getRateOfReturn() {
+        Statistics statistics = new Statistics(List.of(Statistic.THREE, Statistic.FOUR));
+        float result = statistics.getRateOfReturn();
+
+        int totalPrice = Statistic.THREE.getPrice() + Statistic.FOUR.getPrice();
+        float expected = (float) totalPrice / (2 * Lotto.LOTTO_PRICE);
```

Comment:
```suggestion
            float expected = 27.5f;
```

훨씬 테스트의 가독성이 좋아지고, 테스트 하고자함이 분명해지는것 같은데 어떻게 생각하시나요 😄

Reply:
아하. 수식으로 일치 시켜준다고만 생각했는데 이게 훨씬 분명하네요.